udfs/index.md

```
@@ -0,0 +1,416 @@
+            visited, 
+            total + neighbor.amount,
+            reachables
+        );
+    }
+}
+
+function CollectIncreasingAmounts(n) {
+    const visited = [];
+    const reachables = [];
+
+	DFS_IncreasingAmounts(n, visited, n.amount, reachables);
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)